src/SARProcessing.jl

```diff
@@ -3,6 +3,8 @@ module SARProcessing
 using Dates, Statistics
 import XMLDict, Images, ArchGDAL, Polynomials, LinearAlgebra
 
+const LIGHT_SPEED = 299792458.0
```

Comment:
maybe not a super important change for now (given our schedule :D ), but I thought about it twice now: I think it would be more correct to call it SPEED_OF_LIGHT - sorry for this small OCD language comment :D :D

Reply:
We will just change it with our PR so we use the same constant